packages/fast-components-react-base/src/slider/slider.tsx

```diff
@@ -894,6 +908,99 @@ class Slider extends Foundation<SliderHandledProps, SliderUnhandledProps, Slider
         return dragValue;
     };
 
+    /**
+     * Handles touch dragging
+     */
+    private handleUpperThumbTouchStart = (e: React.TouchEvent): void => {
+        this.handleThumbTouchStart(e, SliderThumb.upperThumb);
+    };
+
+    private handleLowerThumbMouseTouchStart = (e: React.TouchEvent): void => {
+        this.handleThumbTouchStart(e, SliderThumb.lowerThumb);
+    };
+
+    private handleThumbTouchStart = (e: React.TouchEvent, thumb: SliderThumb): void => {
+        if (e.defaultPrevented || this.isBusyOrDisabled()) {
+            return;
+        }
+        
+        e.preventDefault();
+        (e.target as HTMLElement).focus();
+        window.addEventListener("touchend", this.handleTouchEnd);
+        window.addEventListener("touchcancel", this.handleTouchEnd);
+        window.addEventListener("touchmove", this.handleTouchMove);
```

Comment:
Just a note: I think we should consider re-evaluating these window events along with #2208 as a breaking change, it seems like we could use forwardRef and localize this event listening to the component.

Not for this PR but as a potential future update.

Reply:
optimizations could be had